nixos/modules/services/misc/wastebin.nix

```
@@ -0,0 +1,158 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.wastebin;
+  inherit (lib)
+    mkEnableOption mkPackageOption mkIf mkOption
+    types mapAttrs isBool getExe boolToString mdDoc optionalAttrs;
+in
+{
+
+  options.services.wastebin = {
+
+    enable = mkEnableOption (mdDoc "Wastenbin pastebin service");
```

```suggestion
    enable = mkEnableOption "Wastenbin pastebin service";
```
https://github.com/NixOS/nixpkgs/pull/287455#discussion_r1489100382
hasn't been addressed properly.
`lib.mdDoc` can be removed *everywhere*
there is no case where it does anything.

I wont comment on every instance of it as there is many, but every single one can be removed :+1:

---

All gone now.